src/snowflake/cli/_app/commands_registration/builtin_plugins.py

```diff
@@ -32,6 +33,7 @@ def get_builtin_plugin_name_to_plugin_spec():
     plugin_specs = {
         "connection": connection_plugin_spec,
         "spcs": spcs_plugin_spec,
+        "migrate-definition": migrate_plugin_spec,
```

Comment:
Not sure "definition" is clear here from context. Also, this feels odd as a top level command. Would it make sense to have some kind of grouping for these "utility" commands? e.g. if we wanted to add a command to validate the PDF file (might be a nice addition to support migration efforts). There would be essentially no good home for this.